### ISSUE TITLE
Installing theme permanently alters Page

### ISSUE THREAD

**Issue by rtpHarry** [Jetski5822/ACMEMedicalTheme#8]
As soon as you install the theme it changes the Page content type to add the Reference Number, Prep Date, Site Area and Tags.

Changing the theme and then disabling the medical theme removes the bing / google settings from the sidebar etc but the page still has these parts attached to them.

Even after you uninstall the theme the parts stay on.

Is it supposed to do this? I can see why it might because it doesn't want to delete your data but then at the same time it the flip side is that each theme can mess your site up just from testing it out?

**Comment by Jetski5822** [comment]
Yeah I understand that. Maybe I could add it as an optional recipe?

**Comment by rtpHarry** [comment]
I don't have much (any) experience with recipes, if you think that would be a cleaner way of doing things then yes :)

With my limited knowledge I am thinking perhaps just move them into their own ContentPart so at least you can remove it via the admin UI after you're finished with the theme?

If recipes can do something slicker then I bow to your superior knowledge!
